README.md

```diff
@@ -1,4 +1,5 @@
 # healthchecks
+[![Build Status](https://travis-ci.org/andela/hc-kalii.svg?branch=ch-travis-integration-154091268)](https://travis-ci.org/andela/hc-kalii)
```

Comment:
@Jak-Sidious the branch should be develop since it's the one being deployed

Reply:
I'm not entirely sure what this comment means, may you please expand on what you're saying so I may attain a clearer picture

Reply:
The link added in the readme.md that displays the badge takes a parameter at the end to indicate which branch of the repo the badge corresponds to. 
Consider: [![Build Status](https://travis-ci.org/andela/hc-kalii.svg?`branch=ch-travis-integration-154091268`)](https://travis-ci.org/andela/hc-kalii)
In the link above, the branch whose build is indicated by the badge is `ch-travis-integration-154091268`.
However, this branch shall inevitably be merged into "develop", at which point the badge shall not be showing. Therefore, the link should have "develop" as the parameter; as in:
`[![Build Status](https://travis-ci.org/andela/hc-kalii.svg?branch=develop)](https://travis-ci.org/andela/hc-kalii)`.